client/hwctl/src/main.rs

```diff
@@ -41,8 +50,8 @@ async fn main() -> Result<()> {
             exit(0);
         }
         Err(e) => {
-            eprintln!("{}", e);
-            exit(1);
+            eprintln!("failed to send certification status request: {}", e);
+            exit(2);
```

Comment:
Are the different exit codes significant? I think it may be cleaner to use `ExitCode::SUCCESS` and `ExitCode::FAILURE` otherwise (from `std::process::exit`).

Also, just to highlight that `exit` isn't defined in this file, I think it would be nice to prefix it with part of the path it's defined in. Not sure how to word it exactly, essentially this:
```rust
use std::process;
// ...
process::exit(...);
```
It may be better to do this in another PR though as I suspect there may be places outside of the current changes which would need to be updated. For this PR, it's probably okay just to follow the existing convention